test/conformance/kuttl/validate/clusterpolicy/cornercases/apply-on-deletion/service.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: podinfo
+  namespace: apply-on-deletion-ns
+spec:
+  clusterIP: 10.43.112.123
+  clusterIPs:
+  - 10.43.112.123
```

Comment:
We have to remove these or else tests will fail if the designed IPs don't fall within the cluster CIDR.

Reply:
Removed!